app/javascript/packs/base.jsx

```diff
@@ -62,6 +65,16 @@ window.Forem = {
   Runtime,
 };
 
+initializePodcastPlayback();
+initializeVideoPlayback();
+InstantClick.on('change', () => {
+  initializePodcastPlayback();
+  initializeVideoPlayback();
+});
+
+// Initialize data-runtime context to the body data-attribute
+document.body.dataset.runtime = window.Forem.Runtime.currentContext();
+
```

Comment:
`window.Forem`/`window.Forem.runtime` is now guaranteed to be available when these function execute since they're called after the initialization took place